utils/config.mlp

```diff
@@ -142,10 +145,14 @@ let print_config oc =
   p "cc_profile" cc_profile;
   p "architecture" architecture;
   p "model" model;
+  p_int "int_size" Sys.int_size;
+  p_int "word_size" Sys.word_size;
   p "system" system;
   p "asm" asm;
   p_bool "asm_cfi_supported" asm_cfi_supported;
   p_bool "with_frame_pointers" with_frame_pointers;
+  p_bool "natdynlink" natdynlink;
```

Comment:
Given that there are various workarounds to determine this already, I'm not sure we should add this to `-config` until its output is really meaningful?